azure/converters/vmss.go

```diff
@@ -163,10 +216,47 @@ func SDKImageToImage(sdkImageRef *compute.ImageReference, isThirdPartyImage bool
 	}
 }
 
-// GetOrchestrationMode returns the compute.OrchestrationMode for the given infrav1.OrchestrationModeType.
-func GetOrchestrationMode(modeType infrav1.OrchestrationModeType) compute.OrchestrationMode {
-	if modeType == infrav1.FlexibleOrchestrationMode {
-		return compute.OrchestrationModeFlexible
+func cgImageRefToImage(id string) infrav1.Image {
```

Comment:
This naming may be confusing with both compute and community gallery abbreviating to `cg`. Would it be better to rename this function, or add a clarifying comment?

Reply:
@willie-yao i added comments to each of the function definitions to further clarify what they're doing.

Reply:
Thanks this looks great! For some reason, I'm unable to resolve these comments.